src/config/src/meta/dashboards/mod.rs

```diff
@@ -166,25 +166,13 @@ impl Dashboard {
     }
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, ToSchema)]
-pub struct Dashboards {
-    pub dashboards: Vec<Dashboard>,
-}
-
```

Comment:
Moved into the `http` module since this is just used as the HTTP response body schema for the `ListDashboards` route